Backend/router/v1/report.ctrl.js

```diff
@@ -55,15 +73,49 @@ const deleteReport = (req, res) => {
       id: reportId
     }
   }).then((result) => {
-    if(result == ""){
-      res.json({msg: "ReportId not exist"});
-    } else {
-      res.status(204).json(result);  //no content
-    };
-  }).catch((err) => {
-    res.json({msg: "error"});
-  });
-}
+    res.status(204).json(result);  //no content
+  }).catch((err) => {});
```

Comment:
Are we intentionally ignoring the error case and not doing anything? If so, please add a comment to justify your decision.

Reply:
I'm so sorry for the trouble, Soo. I modified it in my branch, too.